[Excel] (Custom Functions) Update the JSON metadata reference Options table (#2367)
OfficeDev/office-js-docs-pr: docs/excel/custom-functions-json.md

| Property          | Data type | Required                               | Description |
| :---------------- | :-------- | :------------------------------------- | :---------- |
| `cancelable`      | boolean   | No<br/><br/>Default value is `false`.  | If `true`, Excel calls the `CancelableInvocation` handler whenever the user takes an action that has the effect of canceling the function; for example, manually triggering recalculation or editing a cell that is referenced by the function. Cancelable functions are typically only used for asynchronous functions that return a single result and need to handle the cancellation of a request for data. A function can't use both the `stream` and `cancelable` properties. |

> | `requiresAddress` | boolean   | No <br/><br/>Default value is `false`. | If `true`, your custom function can access the address of the cell that invoked your custom function. To get the address of the cell that invoked your custom function, call the `address` property of the invocation object. For example, if your invocation object is named `invocation`, call `invocation.address` in your custom function. When using `requiresAddress`, the `invocation` parameter must be the last parameter passed to the function. A function can't use both the `stream` and `requiresAddress` properties. |

Reviewer: ```suggestion
| `requiresAddress` | boolean   | No <br/><br/>Default value is `false`. | If `true`, your custom function can access the address of the cell that invoked your custom function. The 'address' property of the [invocation](custom-functions-parameter-options.md#invocation-parameter) parameter contains the address of the cell that invoked your custom function. A function can't use both the `stream` and `requiresAddress` properties. |
```
I added a link to the docs for invocation parameter so we don't need to repeat info like (it's the last parameter) and so on... Need to double-check that this link builds correctly though.
